test-projects/diamond-graph/root/0.1.0/wally.lock

```diff
@@ -0,0 +1,28 @@
+# This file is automatically @generated by Wally.
+# It is not intended for manual editing.
+registry = "test"
+
+[[package]]
+name = "diamond-graph/direct-dependency-a"
+version = "0.1.1"
+dependencies = [["Indirect", "diamond-graph/indirect-dependency-a@0.1.1"]]
+
+[[package]]
+name = "diamond-graph/direct-dependency-b"
+version = "0.1.0"
+dependencies = [["Indirect", "diamond-graph/indirect-dependency-a@0.2.0"]]
+
+[[package]]
+name = "diamond-graph/indirect-dependency-a"
+version = "0.1.1"
+dependencies = []
+
+[[package]]
+name = "diamond-graph/indirect-dependency-a"
+version = "0.2.0"
+dependencies = []
+
+[[package]]
+name = "diamond-graph/root"
+version = "0.1.0"
+dependencies = [["A", "diamond-graph/direct-dependency-a@0.1.1"], ["B", "diamond-graph/direct-dependency-b@0.1.0"]]
```

Comment:
These are already the latest versions of `direct-dependency-a`!

Reply:
```suggestion
# This file is automatically @generated by Wally.
# It is not intended for manual editing.
registry = "test"

[[package]]
name = "diamond-graph/direct-dependency-a"
version = "0.1.0"
dependencies = [["Indirect", "diamond-graph/indirect-dependency-a@0.1.0"]]

[[package]]
name = "diamond-graph/direct-dependency-b"
version = "0.1.0"
dependencies = [["Indirect", "diamond-graph/indirect-dependency-a@0.2.0"]]

[[package]]
name = "diamond-graph/indirect-dependency-a"
version = "0.1.0"
dependencies = []

[[package]]
name = "diamond-graph/indirect-dependency-a"
version = "0.2.0"
dependencies = []

[[package]]
name = "diamond-graph/root"
version = "0.1.0"
dependencies = [["A", "diamond-graph/direct-dependency-a@0.1.0"], ["B", "diamond-graph/direct-dependency-b@0.1.0"]]
```

Reply:
I fixed this by fixing direct-dependency-a to a specific version.

Reply:
I forgot that patches are considered compatible, I'll figure something else out for this in the interm.